README.md

```diff
@@ -1 +1,40 @@
-# Create-mobile-version
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# SET UP MOBILE VERSION
+
+> Description of the project
```

Comment:
- [ ] You should remove the `Description of the project` placeholder text because it makes your README more professional and it is clear that the text is description.

Reply:
"Getting Started" section added as in the README template